java-tools/OsmAndServer/src/main/java/net/osmand/server/api/services/UserdataService.java

```diff
@@ -319,8 +319,10 @@ public ResponseEntity<String> webUserActivate(String email, String token, String
         }
         return registerNewDevice(email, token, TOKEN_DEVICE_WEB, encoder.encode(password));
     }
-    
-	public ResponseEntity<String> webUserRegister(@RequestParam(name = "email", required = true) String email) throws IOException {
+
+	public ResponseEntity<String> webUserRegister(@RequestParam(name = "email", required = true) String email,
+	                                              @RequestParam(name = "lang", required = false) String lang)
```

Comment:
Request for Android here https://github.com/osmandapp/OsmAnd/pull/19058/files

Reply:
android/ios commits are merged